randovania/discord_bot/preset_lookup.py

```diff
@@ -248,7 +243,15 @@ async def _try_get[T](
         return decoder(json.loads(data.decode("utf-8")))
     except Exception as e:
         if not isinstance(e, UnsupportedVersion):
-            logging.exception(f"Unable to process {attachment.filename} from {message}")
+            logging.exception(
+                "Unable to process %s from %s",
```

Comment:
somewhat of a nit, but wouldnt this log be somewhat unreadable if someone sends a message with only the attachment (meaning that `content` would be blank/empty)?